src/main/java/liquibase/ext/opensearch/database/OpenSearchClientDriver.java

```diff
@@ -1,7 +1,7 @@
 package liquibase.ext.opensearch.database;
 
 import liquibase.Scope;
-import liquibase.util.StringUtil;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
I think `liquibase.util.StringUtil` was appropriate here? Why do we need to bring a whole new dependency?

Reply:
`StringUtil#trimToEmpty` is deprecated: `@deprecated use {@link StringUtils#trimToEmpty(String)} instead`
and apache commons is already present, the deprecated API just forwards to it anyway.